### PR TITLE
docs(aws/cloudformation): CloudFormation example from Spinnaker

### PR DESCRIPTION
**Pull request by jorgebee65**
This document will show you how to enable CloudFormation and deploy a simple template (s3 bucket) from Spinnaker.

### CODE REVIEW

File: _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  features:
+    cloudFormation:
+      enabled: true
+```
+You can check your configuration by running `hal deploy apply`.
+
+&nbsp;
+### Halconfig 
+
+Make sure to enable the aws provider and have set at least one region
```
Comment (karlomendozaarmory) [inline _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md]: Add the Hal commands to make this change happen

File: _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+```yaml
```
Comment (karlomendozaarmory) [inline _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md]: leave this as an example of how it should end up looking like

File: _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  defaultRegions:
+  - name: us-west-2
+  defaults:
+    iamRole: BaseIAMRole
+```
+
+&nbsp;
+### CloudFormation permissions in Policy
+
+Another thing to check is the `cloudformation:*` permission in the *"Action"* section inside your **Policy**, like in the following example.
```
Comment (karlomendozaarmory) [inline _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md]: Add a comment that this needs to be done in the AWS console, and what section you should check

File: _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+## Creating your Pipeline
+
+After you've deployed your changes, you should now be able to see the new stages when configuring your pipeline. 
+Simply select the stage, and provide the values: example below: 
+
+![](/images/cloudformation-pipeline1.png)
+
+In this exaple we are going to deploy a simple s3 bucket with the follow template, (Copy and paste in the **Text Source** section)
+
```
Comment (karlomendozaarmory) [inline _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md]: Add a comment about the name having to be different for the cloud formation template

File: _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+   - AWS
+description: How to enable CloudFormation and deploy a simple template.
+type: Document
+---
+
+## Background
+
+This document will show you how to enable the CloudFormation stage.  This document also assumes that you have configured an AWS account.  If you need more instructions on configuring your AWS account please refer to either [Deploying to AWS from Spinnaker (using IAM instance roles)](https://docs.armory.io/spinnaker-install-admin-guides/add-aws-account-iam/) or [Deploying to AWS from Spinnaker (using IAM credentials)](https://docs.armory.io/spinnaker-install-admin-guides/add-aws-account/) 
+
+## Preconditions
```
Comment (karlomendozaarmory) [inline _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md]: Doesn't this need to run the terraform scripts to work? there is no mention of the scripts nowhere.

File: _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md (hunk 1 above)
Comment (jorgebee65) [reply _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md]: Hal commands added

File: _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md (hunk 2 above)
Comment (jorgebee65) [reply _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md]: done

File: _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md (hunk 3 above)
Comment (jorgebee65) [reply _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md]: also put the reference to the another doc

File: _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md (hunk 4 above)
Comment (jorgebee65) [reply _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md]: Note added, for the "Stack name" should be unique.

File: _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md (hunk 5 above)
Comment (jorgebee65) [reply _posts/aws/2019-07-16-AWS-CloudFormation-HowTo-enable.md]: The terraform scripts are for other purposes: To create IAM Roles and Policies. In this article we are assuming the user already have configured an AWS account.

**Pull request merged**